src/yugabyte-bash-common.sh

```diff
@@ -320,6 +322,16 @@ log_with_color() {
   log "$log_color$*$NO_COLOR"
 }
 
+warn() {
+  local stack_idx0=${yb_log_skip_top_frames:-0}
+  local stack_idx1=$(( stack_idx0 + 1 ))
+
+  # shellcheck disable=SC2048,SC2086
+  echo -e "[$( get_timestamp )" \
+       "${BASH_SOURCE[$stack_idx1]##*/}:${BASH_LINENO[$stack_idx0]}" \
+       "${FUNCNAME[$stack_idx1]}]" ${RED_COLOR} $* ${NO_COLOR} >&2
```

Comment:
Should we check if the terminal supports colors and avoid using colors if it does not?
(That was my biggest issue with various kinds of terminal colors in log messages.)

Reply:
Hm, yes, that is a legit concern.  And I am not sure of a very clean way to that.